### PR TITLE
update amazon sdk 1.10.60

### PR DESCRIPTION
**Pull request by wstrucke**
update Amazon SDK to latest

### CODE REVIEW

**Comment by wstrucke** [comment]
ERROR: Step ‘Publish JUnit test result report’ failed: No test report files were found. Configuration error?

Can someone provide guidance on what I should add to get this build step to work properly?

File: src/main/scala/com/netflix/edda/aws/S3CurrentDatastore.scala [hunk 1]
```diff
@@ -63,11 +63,9 @@ class S3CurrentDatastore(val name: String) extends Datastore {
   // of sync with the datastore
   val writeDynamo = {
     val client = new AwsClient(account).dynamo
-    val config = new com.amazonaws.ClientConfiguration().withMaxErrorRetry(0)
-    client.setConfiguration(config)
```
Comment (brharrington) [inline src/main/scala/com/netflix/edda/aws/S3CurrentDatastore.scala]: What was the rationale for removing this?

**Comment by brharrington** [comment]
Most of our builds have moved to travis. I'll look into getting this one switched over and building cleanly.

File: src/main/scala/com/netflix/edda/aws/S3CurrentDatastore.scala (hunk 1 above)
Comment (wstrucke) [inline src/main/scala/com/netflix/edda/aws/S3CurrentDatastore.scala]: Edda would not build with the new version of the SDK with that.  Removing it did not seem to have an affect on the application, though I am only using Mongo and not S3.

**Pull request merged**